apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/withdrawInvoiceButton.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable unicorn/no-null */
+import { useRef } from 'react'
+import { useForm } from 'react-hook-form'
+import { useMutation } from 'urql'
+
+import { FragmentType, graphql, useFragment } from '../../../../../../frontend/generated/gql'
+import { InvoiceSendInput } from '../../../../../../frontend/generated/gql/graphql'
+
+const WithdrawInvoiceMutationDocument = graphql(`
```

Comment:
```suggestion
const InvoiceWithdrawMutationDocument = graphql(`
```
Again, for consistency. Same for `withdrawInvoice`, `WithdrawInvoiceButtonProps`, `SendInvoiceMutationDocument`, `sendInvoice` ...